app/javascript/app/components/sectors-agriculture/countries-context/land-area/land-area-component.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LandAreaProvider from 'providers/agriculture-land-area-provider';
+import { SunburstChart, Card, Tag, Icon } from 'cw-components';
+import infoIcon from 'assets/icons/info';
+import Tooltip from './tooltip';
+
+import styles from './land-area-styles.scss';
+
+const cardTheme = {
+  card: styles.card,
+  data: styles.data
+};
+
+const LandArea = ({
+  selectedCountry,
+  selectedYear,
+  chartData,
+  chartColors,
+  chartConfig,
+  firstLevelLegend,
+  handleInfoBtnClick
+}) => (
+  <div className={styles.container}>
+    <Card
+      key={'Share in Land Area'}
+      title={'Share in Land Area'}
+      theme={cardTheme}
+    >
+      <div className={styles.cardContainer}>
+        <div className={styles.header}>
+          <div className={styles.title}>
+            Share of different types of agricultural land as a percentage of
+            total land area in a country. Depending on agricultural intensity,
+            countries devote a different amount of land to agriculture.
+          </div>
+          <Icon
+            icon={infoIcon}
+            theme={{ icon: styles.cardInfoIcon }}
+            onClick={handleInfoBtnClick}
+          />
+        </div>
+        <div className={styles.chartContainer}>
+          <div className={styles.legend}>
+            {firstLevelLegend &&
+              firstLevelLegend.map(q => (
+                <React.Fragment>
+                  <Tag
```

Comment:
You can move component `<Tag />`  to separate method `renderTag() { ... }` because the only difference between tag and nested tag is theme key which you can pass ;)

Reply:
👍